Snail/Snail+Combine/DemandBuffer.swift

```diff
@@ -0,0 +1,76 @@
+//  Copyright © 2021 Compass. All rights reserved.
+
+import Combine
+import Foundation
+
+@available(iOS 13.0, *)
+class DemandBuffer<S: Combine.Subscriber> {
```

Comment:
Nice! This is directly taken from RxSwift project right? If so, lets add a link to that source here for future references.

Reply:
For sure! Forgot to get the original file as I was playing around with this one. I'll add the original in w/ attribution.